vocode/streaming/transcriber/assembly_ai_transcriber.py

```diff
@@ -51,7 +51,10 @@ def terminate(self):
         self._ended = True
 
     def get_assembly_ai_url(self):
-        return ASSEMBLY_AI_URL + f"?sample_rate={self.transcriber_config.sampling_rate}"
+        sample_rate = 16000
```

Comment:
we can't hard-code sampling rate - the rate is a variable in `AssemblyAITranscriberConfig`. If AssemblyAI works worse at different sampling rates, maybe we need to use `audioop` to change the sampling rate on the way in.